rca/static_src/javascript/programmes/programmes.api.js

```diff
@@ -1,17 +1,33 @@
 import PropTypes from 'prop-types';
 
-import { programmePage } from './programmes.types';
-
-const PROGRAMME_TYPE = 'programmes.ProgrammePage';
-const PROGRAMME_FIELDS = [
-    'degree_level',
-    'programme_description_subtitle',
-    'pathway_blocks',
-    'hero_image_square',
-];
+import {
+    PROGRAMME_PAGE_TYPE,
+    SHORT_COURSE_PAGE_TYPE,
+    programmePage,
+} from './programmes.types';
 
 const WAGTAIL_API_ENDPOINT = '/api/v3/pages';
 
+/**
+ * The programmes API queries multiple page types, with different fields for each.
+ * This array also determines the order in which the instances are displayed.
+ */
+const listedPageTypes = [
+    {
+        type: PROGRAMME_PAGE_TYPE,
+        fields: [
+            'summary',
+            'hero_image_square',
+            'degree_level',
+            'pathway_blocks',
+        ],
+    },
+    {
+        type: SHORT_COURSE_PAGE_TYPE,
+        fields: ['summary', 'hero_image_square'],
+    },
+];
```

Comment:
Note this is the array that might need reordering depending on feedback on the ticket.

We may also need to implement client-side sorting if we want the two types to be mixed – that would need be implemented right after `const items = results.reduce((all, res) => all.concat(res.items), []);`, being careful to only do this for **filtering** responses rather than search results (which should be ordered by relevance server-side).